gradle.properties

```diff
@@ -1,8 +1,15 @@
 # Daemon’s heap size
-org.gradle.jvmargs=-Xms1g -Xmx1g -XX:MaxPermSize=512m -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xms1g -Dkotlin.daemon.jvm.options\="-Xmx1536M" -Xmx1536M -XX\:MaxPermSize\=512m -Dfile.encoding\=UTF-8
 
 # Parallel execution
 org.gradle.parallel=true
 
 # Cacheable unit tests
 android.testConfig.useRelativePath = true
+
+# Migrating to AndroidX - AGP 3.6.x
+android.useAndroidX=true
+#android.enableJetifier=true - causing SHA1 digest error
```

Comment:
similar issue https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/455